pkg/reconciler/apiserversource/resources/receive_adapter.go

```diff
@@ -45,6 +45,7 @@ type ReceiveAdapterArgs struct {
 	Source        *v1.ApiServerSource
 	Labels        map[string]string
 	SinkURI       string
+	CACerts       *string
```

Comment:
This variable is never set anywhere, we need to set it here https://github.com/knative/eventing/blob/66e8257b9c834f4456795562b38720e9de01ac0e/pkg/reconciler/apiserversource/apiserversource.go#L121C15-L125 when creating the receiver adapter.

We can pass `sinkAddr` to `createReceiveAdapter` and then we can use it like this:

```
	adapterArgs := resources.ReceiveAdapterArgs{
		Image:         r.receiveAdapterImage,
		Source:        src,
		Labels:        resources.Labels(src.Name),
		CACerts:       sinkAddr.CACerts,
		SinkURI:       sinkAddr.URL.String(),
		Configs:       r.configs,
		Namespaces:    namespaces,
		AllNamespaces: allNamespaces,
	}
```